lib/rocketfuel_api/service.rb

```diff
@@ -19,12 +19,13 @@ def name
     end
   end
 
-  def plural_name
-    name + 's'
-  end
-
   def uri_suffix
-    plural_name
+    file_path = File.join(RocketfuelApi.root, 'lib/config/endpoints_for_services.yaml')
```

Comment:
You can write this as `RocketfuelApi.root.join('lib', 'config', 'endpoints_for_services.yaml')`.